index.html

```diff
@@ -0,0 +1,292 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Riyadh Season</title>
+    <link rel="stylesheet" href="styleCap/reset.css" />
+    <link rel="stylesheet" href="styleCap/capstone.css" />
+    <script src="https://kit.fontawesome.com/04970764d6.js" crossorigin="anonymous"></script>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="about/about.html">
+  </head>
+  <body>
+  
+    <header class="header-background">
+
+      <nav class="mobile-nav">
+        <div class="hamburger-menu">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+        <ul class="nav-menu">
+          <li class="nav-item">
+            <a href="index.html" class="nav-link text-muted">Home</a>
+          </li>
+          <li>
+            <a href="about/about.html" class="nav-link text-muted">About</a>
+          </li>
+          <li>
+            <a href="#" class="nav-link text-muted">Program</a>
+          </li>
+          <li>
+            <a href="#" class="nav-link text-muted">Sponser</a>
+          </li>
+          <li>
+            <a href="#" class="nav-link text-muted">Join</a>
+          </li>
+          <li>
+            <a href="#" class="nav-link text-muted">News</a>
+          </li>
+        </ul>
+      </nav>
+
+      <nav class="first-title">
+      <ul class="title-black">
+        <li>
+          <a href="#" class=""><i class="fab fa-facebook-f"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fab fa-twitter"></i></a>
+        </li>
+        <li>
+          <a href="#">English</a>
+        </li>
+        <li>
+          <a href="#">My Page</a>
+        </li>
+        <li>
+          <a href="#">Logout</a>
+        </li>
+      </ul>
+    </nav>
+
+    <nav class="second-title">
+      <div class="logo">
+        <a href="index.html"
+          ><img
+            class="logo-img"
+            src="https://www.arabnews.com/sites/default/files/main-image/2019/10/04/1783706-1871065082.jpg"
+            alt="riyadh season logo"
+        /></a>
+      </div>
+      <ul class="title-white">
+        <li>
+          <a href="about/about.html" class="nav-link text-muted">About</a>
+        </li>
+        <li>
+          <a href="#" class="nav-link text-muted">Program</a>
+        </li>
+        <li>
+          <a href="#" class="nav-link text-muted">Sponser</a>
+        </li>
+        <li>
+          <a href="#" class="nav-link text-muted">Join</a>
+        </li>
+        <li>
+          <a href="#" class="nav-link text-muted">News</a>
+        </li>
+      </ul>
```

Comment:
In your design, the unordered list is centered while if compared to the behance design, it seems more on the right. See the comarison below 👇🏻 

- [ ] I suggest that the position of the ul is changed from center to extend to the right.

| Your design | Behance design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/4247740/153692758-72434332-8ca7-4277-a1a3-19f1896b3850.png) | ![image](https://user-images.githubusercontent.com/4247740/153692833-dc31662c-1c78-4a0d-b67f-914efef2d819.png) |

Reply:
:heavy_check_mark: